app/services/pii/cacher.rb

```diff
@@ -0,0 +1,31 @@
+module Pii
+  class Cacher
+    def initialize(user, user_session)
+      @user = user
+      @user_session = user_session
+    end
+
+    def save(password, profile = user.active_profile)
+      return unless profile
+      decrypted_pii = profile.decrypt_pii(password)
+      pii_json = decrypted_pii.to_json
+      encrypted_pii = encryptor.encrypt(pii_json)
+      user_session[:encrypted_pii] = encrypted_pii
+    end
```

Comment:
it's hard to see that in this PR because there is no actual encryption happening. All the crypto was yanked out to simplify the review process.

In a subsequent PR, the passphrase will actually be used for encryption, and there will be different encryption methods used for storage in the db vs storage in the session.